tests/autoyast/installation.pm

```diff
@@ -102,6 +104,23 @@ sub verify_timeout_and_check_screen {
     return check_screen $needles, $check_time;
 }
 
+sub wait_boot_on_local_disk {
+   send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
```

Comment:
Once again, we have 100% same code in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L370 
Could you please put this method in opensusebasetest and call it in L#374 and in this module.

Reply:
Updated code following your comments, please have a look.

Reply:
LGTM, please, resolve conflicts and we can merge.